Bottom: 75110ef0f4cf2076f1f4aa62b929796f1435d60e
Top:    e16e386391ab3d9f1128f422c81560d225c291ee
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-16 16:24:21 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 0954882..23cca2d 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -124,7 +124,6 @@ a zoom in/out.
 #include "gui-icons-misc.data"
 #include "gui-trackball.h"
 #include "snavi.h"
-#include "gui-trackball.h"
 
 #ifdef HAVE_LIBDMALLOC
 #include <dmalloc.h>
